ukol_03/sifra.py

```diff
@@ -0,0 +1,14 @@
+plain_text = input("Please, insert your message to encrypt: ")
+shift = int(input("Please, insert shift: "))
+result = ""
+
+for character in plain_text:
+    if character == " ": 
+        result = result + " "
+    else: 
+        x = ord(character)
+        result += chr(x + shift)
+
+print(result)
+
+lets_continue = input("Do you want to insert another message to encrypt? ")
```

Comment:
Tomuto troška nechápem, nakoľko podľa popisu by to malo nejako reštartovať program, ale nespraví to nič... :cry: Ak by si nejakú takúto funkcionalitu chcela docieliť je možnosť vyčleniť to do funkcie (túto tému sme ešte nemali...), alebo to celé obaliť do nejakého cyklu... Tak₌ze buď túto časť odstráň, alebo nejak prerob :smile: